src/main/kotlin/no/nav/familie/ef/sak/brev/dto/FrittståendeBrevKategori.kt

```diff
@@ -11,5 +11,6 @@ enum class FrittståendeBrevKategori(val frittståendeBrevType: FrittståendeBre
     INNHENTING_AV_KARAKTERUTSKRIFT_UTVIDET_PERIODE(frittståendeBrevType = FrittståendeBrevType.INNHENTING_AV_KARAKTERUTSKRIFT_UTVIDET_PERIODE),
     BREV_OM_SVARTID_KLAGE(frittståendeBrevType = FrittståendeBrevType.BREV_OM_SVARTID_KLAGE),
     BREV_OM_FORLENGET_SVARTID(frittståendeBrevType = FrittståendeBrevType.BREV_OM_FORLENGET_SVARTID),
-    BREV_OM_FORLENGET_SVARTID_KLAGE(frittståendeBrevType = FrittståendeBrevType.BREV_OM_FORLENGET_SVARTID_KLAGE)
+    BREV_OM_FORLENGET_SVARTID_KLAGE(frittståendeBrevType = FrittståendeBrevType.BREV_OM_FORLENGET_SVARTID_KLAGE),
+    INFORMASJONSBREV_TRUKKET_SØKNAD(frittståendeBrevType = FrittståendeBrevType.INFORMASJONSBREV)
```

Comment:
Mulig denne skal få en egen type i kontrakter for å få eget navn i dokumentoversikten. Tar en sjekk med Mirja før merge.